packages/google-vertex/src/google-vertex-provider.ts

```diff
@@ -43,24 +43,22 @@ Your Google Vertex project. Defaults to the environment variable `GOOGLE_VERTEX_
   project?: string;
 
   /**
- Optional. The Authentication options provided by google-auth-library.
-Complete list of authentication options is documented in the
-GoogleAuthOptions interface:
-https://github.com/googleapis/google-auth-library-nodejs/blob/main/src/auth/googleauth.ts.
+   * Headers to use for requests. Can be:
+   * - A headers object
+   * - A Promise that resolves to a headers object
+   * - A function that returns a headers object
+   * - A function that returns a Promise of a headers object
    */
-  googleAuthOptions?: VertexInit['googleAuthOptions'];
```

Comment:
Removing the auth options will be breaking. Can we keep them for the node.js version?

Reply:
Oh yes, this was an oversight. We should be able to keep them, I'll look in the morning.

Reply:
I had already preserved them, they are here: https://github.com/vercel/ai/blob/ac19c52d9ff5a6900919ac31fd45d25dd15592b9/packages/google-vertex/src/google-vertex-provider-node.ts#L17

The removal here is from the shared base provider, which makes sense to do.